lazyflow/operators/ioOperators/opExportSlot.py

```diff
@@ -236,40 +240,46 @@ def run_export(self):
         else:
             mkdir_p( PathComponents(self.ExportPath.value).externalDirectory )
             export_func()
-    
-    def _export_hdf5(self, compress=False):
+
+    def _export_h5n5(self, compress=False):
         self.progressSignal( 0 )
 
-        # Create and open the hdf5 file
+        # Create and open the hdf5/n5 file
         export_components = PathComponents(self.ExportPath.value)
         try:
-            os.remove(export_components.externalPath)
+            if export_components.extension in OpStreamingH5N5Reader.N5EXTS:
+                if os.path.isdir(export_components.externalPath):
+                    shutil.rmtree(export_components.externalPath)
+                else:
+                    os.remove(export_components.externalPath)
+            else:
+                os.remove(export_components.externalPath)
```

Comment:
I'm a bit confused by this - sorry. So, we have two possibilities here, right?

a) `n5` file: -> delete a folder
b) `hdf5` file: -> delete a file

but in a) you also seem to be expecting files as external paths? Or is it for the case that, for whatever reason someone actually had a single file there with the `.h5` extension?

Reply:
Yep, thats sloppy. I removed the else-block